pyaedt/q3d.py

```diff
@@ -72,26 +51,39 @@ def __exit__(self, ex_type, ex_value, ex_traceback):
         if ex_type:
             exception_to_desktop(self, ex_value, ex_traceback)
 
+
 class Q3d(QExtractor, object):
-    """Q3D Object
+    """Q3D application interface.
+
+    Allows you to create an instance of Q3D and link to an
+    existing project or create a new one.
 
     Parameters
     ----------
     projectname : str, optional
-        Name of the project to select or the full path to the project or AEDTZ archive to open. 
-        The default is ``None``. If ``None``, try to get an active project and, if no projects are present, 
-        create an empty project.
+        Name of the project to select or the full path to the project
+        or AEDTZ archive to open.  The default is ``None``. If
+        ``None``, try to get an active project and, if no projects are
+        present, create an empty project.
     designname : str, optional
-        Name of the design to select. The default is ``None``. If ``None``, try to get an active design and, 
-        if no designs are present, create an empty design.
+        Name of the design to select. The default is ``None``. If
+        ``None``, try to get an active design and, if no designs are
+        present, create an empty design.
     solution_type : str, optional
-        Solution type to apply to design. The default is ``None``. If ``None``, the default type is applied.
+        Solution type to apply to design. The default is ``None``. If
+        ``None``, the default type is applied.
     setup_name : str, optional
-        Name of the setup to use as the nominal. The default is ``None``. If ``None``, the active setup 
-        is used or nothing is used.
-    
-    Returns
-    -------
+        Name of the setup to use as the nominal. The default is
+        ``None``. If ``None``, the active setup is used or nothing is
+        used.
+
+    Examples
+    --------
+    Create an instance of ``Q3d`` and connect to an existing Q3D
+    design or create a new Q3D design if one does not exist.
+
+    >>> from pyaedt import Q3d
+    >>> app = Q3d()
 
```

Comment:
For some reason, I though you said not to put a blank line after the closing doc string """. Should I or shouldn't I. I've been removing them.

Reply:
Remove them.  I goofed up here.

Reply:
```suggestion
```